tests/integration/BUILD

```diff
@@ -22,6 +22,7 @@ genquery(
         "@testonly_testing//:com_google_code_findbugs_jsr305",
         "@testonly_testing//:com_google_auto_value_auto_value_annotations_1_6_3",
         "@testonly_testing//:com_google_auto_value_auto_value_annotations",
+        "@testonly_testing//:pin",
```

Comment:
The pinned repo should not have the pin build target.

Reply:
`testonly_testing` isn't pinned (no maven_install_json), so it should have a `:pin` target to create the initial JSON. right? https://github.com/bazelbuild/rules_jvm_external/blob/7f29105eb4ac2c1a1582197337656d6e59b0acb9/WORKSPACE#L226

Reply:
Oops, you're right.